constants.js

```diff
@@ -40,7 +40,7 @@ let _inappPreviewHost = '';
 
 switch ( origin ) {
     case 'https://local.webaverse.com': {
-        _inappPreviewHost = 'https://local.webaverse.online';
+        _inappPreviewHost = `https://local.webaverse.com:${window.location.port}`;
```

Comment:
This is not legal. The iframe must be on a different tld+1 to get a new render process.

Reply:
ahh, I remember you mentioning the new render thread. It makes sense because it thinks it's a new website.  

This was initially set to `.online` with no port and if you ran it off 3000, it would never load. I thought it was the wrong URL, but it was just missing the port. Reverted to `.online` but kept the port so that it'll work for any port when running local.